pareto/examples/multiobjective_optimization/seismicity_vs_cost_MOO.ipynb

```diff
@@ -0,0 +1,531 @@
+   "id": "f42b8b21-8f1e-4b22-97e3-e0f8a4cc1ae9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "options = {\n",
+    "    \"deactivate_slacks\": True,\n",
+    "    \"scale_model\": False,\n",
+    "    \"scaling_factor\": 1000,\n",
+    "    \"running_time\": 200,\n",
+    "    \"gap\": 0,\n",
+    "}\n",
+    "\n",
+    "results_min_cost = solve_model(model=model, options=options)\n",
+    "\n",
+    "assert results_min_cost.solver.termination_condition == TerminationCondition.optimal\n",
+    "assert results_min_cost.solver.status == SolverStatus.ok\n",
+    "\n",
+    "with nostdout():\n",
+    "    feasibility_status = is_feasible(model)\n",
+    "\n",
+    "assert feasibility_status"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed091ed2-f38c-4025-b9bc-6b969acb26e2",
+   "metadata": {},
+   "source": [
+    "We investigate the value of the objective function (minimum cost) and the corresponding subsurface risk:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6e5b284-d42e-4346-b353-11a4115adb00",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_cost_obj = model.v_Z.value\n",
+    "min_cost_risk = model.v_Z_SubsurfaceRisk.value\n",
+    "\n",
+    "print(\"Minimum cost model\")\n",
+    "print(\"------------------\")\n",
+    "print(f\"Minimum cost achievable: {min_cost_obj} {model.v_Z.get_units()}\")\n",
+    "print(\n",
+    "    f\"Risk incurred with minimum cost: {min_cost_risk} {model.v_Z_SubsurfaceRisk.get_units()}\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a6b768c-691d-4909-bee7-8b4ef1643b7b",
+   "metadata": {},
+   "source": [
+    "It's straightforward to alter the model to minimize subsurface risk instead of cost, and then reoptimize:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8718cb3-6546-4469-8be5-fe22a560480b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change the objective function in the model from minimum cost to minimum subsurface risk\n",
+    "set_objective(model, Objectives.subsurface_risk)\n",
+    "\n",
+    "results_min_risk = solve_model(model=model, options=options)\n",
+    "\n",
+    "assert results_min_risk.solver.termination_condition == TerminationCondition.optimal\n",
+    "assert results_min_risk.solver.status == SolverStatus.ok\n",
+    "\n",
+    "with nostdout():\n",
+    "    feasibility_status = is_feasible(model)\n",
+    "\n",
+    "assert feasibility_status"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfa3a699-dd91-43d6-b5a3-89512adb6552",
+   "metadata": {},
+   "source": [
+    "We investigate the value of the objective function (minimum subsurface risk) and the corresponding cost:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "880b8561-b5f9-4470-acc0-fece09e40a03",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_risk_obj = model.v_Z_SubsurfaceRisk.value\n",
```

Comment:
`model.v_Z_SubsurfaceRisk.value + model.subsurface.y_dist`

Reply:
I changed the subsurface risk objective function so that this suggestion is not necessary.